modules/gles3/functional/es3fNegativeTextureApiTests.cpp

```diff
@@ -1452,7 +1461,7 @@ void NegativeTextureApiTests::init (void)
 			glTexSubImage2D(GL_TEXTURE_2D, 0, 0, 0, 4, 4, GL_RGBA_INTEGER, GL_UNSIGNED_INT, 0);
 			expectError(GL_INVALID_OPERATION);
 			glTexSubImage2D(GL_TEXTURE_2D, 0, 0, 0, 4, 4, GL_RGB, GL_FLOAT, 0);
-			expectError(GL_INVALID_OPERATION);
+			expectError(isES ? GL_INVALID_OPERATION : GL_NO_ERROR);
```

Comment:
There is no requirement on GL4.5 for implementations to validate that the pointer sent to glTexSubImage* is valid. This can cause conformant implementations to crash and should be skipped for GL4.5 runs.

Reply:
Done